### PR TITLE
Fix issues with date representation

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team, 

I do not know why the tests are passing in CI, but they started to fail in my local environment and it was because I was using Dates in _Reports.test.js_'s assertions.

I fixed that, and I saw that besides that, we were rendering a wrong date because _FormattedDate_ uses current timezone by default. 

In order to avoid this problem by the moment **we should use _timeZone_ parameter** in this way:

```
<FormattedDate value={domainSelected.verified_date} timeZone="UTC" />
``` 

I have also removed some redundant assertions.

Could you kindly review it?

### CODE REVIEW

File: src/components/Reports/Reports.test.js [hunk 1]
```diff
@@ -46,13 +49,8 @@ describe('Reports page', () => {
       </DopplerIntlProvider>,
     );
 
-    await wait(() => getByText(fakeData[0].verified_date));
-
-    const verifiedDate = getByText(fakeData[0].verified_date);
-    const domain = getByText(fakeData[1].name);
-
-    expect(verifiedDate).toBeDefined();
```
Comment (fcoronelmakingsense) [inline src/components/Reports/Reports.test.js]: i didn't know no need expect in a test. I always saw it in the examples

File: src/components/Reports/ReportsFilters/ReportsFilters.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const ReportsFilters = ({
             {domainSelected ? (
               <span className="verified--domain">
                 <FormattedMessage id="reports_filters.verified_domain" />{' '}
-                <FormattedDate value={domainSelected.verified_date} />
+                <FormattedDate value={domainSelected.verified_date} timeZone="UTC" />
```
Comment (fcoronelmakingsense) [inline src/components/Reports/ReportsFilters/ReportsFilters.js]: 👍

File: src/components/Reports/Reports.test.js (hunk 1 above)
Comment (cbernat) [inline src/components/Reports/Reports.test.js]: Why did you remove this expects? Shouldn't all tests have at least one expect?

Comment (andresmoschini) [reply src/components/Reports/Reports.test.js]: You said me that _getByText_ fails if the literal is not found, so, expected is redundant, right?

Comment (fcoronelmakingsense) [reply src/components/Reports/Reports.test.js]: yes but more clear with expect. for me at least 😄

Comment (andresmoschini) [reply src/components/Reports/Reports.test.js]: I removed because _getByText_ verify that the expected text exists, for example:

```html
<div>2018-01-01</div>
```

```js
const verifiedDate = getByText('2020-05-05'); // It breaks here.
expect(verifiedDate).toBeDefined(); // this line is never executed.
```

---

```html
<div>2018-01-01</div>
```

```js
const verifiedDate = getByText('2018-01-01'); // It pass, so verifiedDate is defined
expect(verifiedDate).toBeDefined(); // this line will pass
```

Comment (andresmoschini) [reply src/components/Reports/Reports.test.js]: Mmmm, in general, I think that it is not a good idea having redundant code, in my opinion, it tends to be dangerous. 

But, if you want I can add these lines again, by the moment it is not so important (for me)

Comment (andresmoschini) [reply src/components/Reports/Reports.test.js]: @fcoronelmakingsense @cbernat let me know your opinion, in order to update it (or not) before merging.

Comment (fcoronelmakingsense) [reply src/components/Reports/Reports.test.js]: i'm already put my opinion but it's works the 2 ways

Comment (cbernat) [reply src/components/Reports/Reports.test.js]: Yes it is redundant, and yes it is clearer with the expect. Both ways work for me. I think is an issue of how we want to work.
